streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1080,11 +1108,11 @@ public Sensor addLatencySensor(String scopeName, String entityName, String opera
             String metricGroupName = "stream-" + scopeName + "-metrics";
 
             // first add the global operation metrics if not yet, with the global tags only
-            Sensor parent = metrics.sensor(sensorNamePrefix + "." + scopeName + "-" + operationName);
+            Sensor parent = metrics.sensor(sensorNamePrefix + "." + scopeName + "-" + operationName, recordLevel);
             addLatencyMetrics(metricGroupName, parent, "all", operationName, this.metricTags);
 
             // add the store operation metrics with additional tags
-            Sensor sensor = metrics.sensor(sensorNamePrefix + "." + scopeName + "-" + entityName + "-" + operationName, parent);
+            Sensor sensor = metrics.sensor(sensorNamePrefix + "." + scopeName + "-" + entityName + "-" + operationName, recordLevel, parent);
```

Comment:
This is not introduced in this patch, but we could fix it together:

in line 1133 below

```
Sensor sensor = metrics.sensor(sensorNamePrefix + "-" + entityName + "-" + operationName);
```

We'd better change it to

```
Sensor sensor = metrics.sensor(sensorNamePrefix + "." + entityName + "-" + operationName);
```